src/Domain/Shared/ValueObject/AbstractUuid.php

```diff
@@ -49,9 +49,9 @@ public static function fromInteger(string $integer): self
     /**
      * @param int|string $node
      *
-     * @return static
-     *
      * @throws \Exception
+     *
+     * @return static
```

Comment:
`static` and `self` are too different things. I really prefer avoid return type if this is a problem for phpstan